nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -286,6 +286,9 @@ def get_param_groups(self) -> Dict[str, List[Parameter]]:  # pylint: disable=no-
         """
         return {}
 
+    def at_train_end(self, **kwargs: Any) -> Optional[Any]:  # pylint: disable=unused-argument disable=no-self-use
+        """Called at end of training for optional datamanager outputs."""
+
```

Comment:
This is never called, I worry that users will override this function and expect it to be called.